examples/viewer.py

```diff
@@ -53,24 +53,26 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
         )
 
         # Compute environment camera resolution based on the number of environments to render in the window.
-        surface_size: tuple(int, int) = (
+        window_size: mn.Vector2 = (
             self.sim_settings["window_width"],
             self.sim_settings["window_height"],
         )
-        grid_size: tuple(int, int) = ReplayRenderer.environment_grid_size(self.num_env)
-        camera_resolution: tuple(int, int) = (
-            surface_size[0] / grid_size[0],
-            surface_size[1] / grid_size[1],
-        )
-        self.sim_settings["width"] = camera_resolution[0]
-        self.sim_settings["height"] = camera_resolution[1]
 
         configuration = self.Configuration()
         configuration.title = "Habitat Sim Interactive Viewer"
-        configuration.size = surface_size
+        configuration.size = window_size
         Application.__init__(self, configuration)
         self.fps: float = 60.0
 
+        # Compute environment camera resolution based on the number of environments to render in the window.
+        grid_size: tuple(int, int) = ReplayRenderer.environment_grid_size(self.num_env)
+        camera_resolution: mn.Vector2 = (
+            self.framebuffer_size[0] / grid_size[0],
+            self.framebuffer_size[1] / grid_size[1],
+        )
```

Comment:
`environment_grid_size()` returns a Vector2i too, so you can work with it as a vector as well -- just cast to a `Vector2` first to get a floating-point division instead of an integer one:

```suggestion
        grid_size: mn.Vector2i = ReplayRenderer.environment_grid_size(self.num_env)
        camera_resolution: mn.Vector2 = mn.Vector2(self.framebuffer_size) / mn.Vector2(grid_size)
```